runtime/moonbase/tests/xcm_mock/parachain.rs

```diff
@@ -815,6 +869,8 @@ impl pallet_xcm_weight_trader::Config for Runtime {
 	type XcmFeesAccount = XcmFeesAccount;
 	#[cfg(feature = "runtime-benchmarks")]
 	type NotFilteredLocation = RelayLocation;
+	#[cfg(feature = "runtime-benchmarks")]
+	type AssetCreator = EvmForeignAssets;
```

Comment:
Isn't `pallet_treasury::ArgumentsFactory` enough? Why should we need this?

Reply:
The only thing the `ArgumentsFactory` does it generate the arguments that will be passed to the benchmark instance. For example, we are going to use `NativeOrWithId::WithId(1)` and not the native asset. But it does not setup anything that might be needed for that scenario to work (like creating the asset). The `ensure_successful` methods are the ones in charge of the setup.